### PR TITLE
Disable code scanning alerts scorecards.yml

### PR DESCRIPTION
**Pull request by joycebrum**
Related to https://github.com/systemd/systemd/issues/25042#issuecomment-1564849664

It is possible to temporarily disable the upload to code scanning if the "false positives" are too bothering. We can enable it again once many of these false positives got addressed.

### CODE REVIEW

**Comment by bluca** [comment]
What does this disable, precisely?

**Comment by evverx** [comment]
It prevents scorecard from uploading its alerts to the security dashboard. scorecard isn't compatible with how SARIF is supposed to work in general (and that's why it was broken recently by a minor SARIF update). Alerts produced due to https://github.com/ossf/scorecard/issues/3156 can't be dismissed properly because that's how the action, scorecard, OSV and OSS-Fuzz work and so on.

The same info can be found at https://securityscorecards.dev/viewer/?platform=github.com&org=systemd&repo=systemd (once it's officially released).

**Comment by evverx** [comment]
I'm not sure it should be commented out. I think it should be removed. I'm not sure it should upload artifacts and keep them for 5 days either. Those SARIF files aren't helpful in terms of debugging because the "verbose" mode hasn't be implemented yet. I think the part where the PAT is mentioned can be removed too. That PAT is dangerous: https://github.com/ossf/scorecard-action#personal-access-token-pat-requirements-and-risks and I don't think anyone is going to ever use it here (or anyone else for that matter).

@jamacku could you take a look?

**Review by jamacku — CHANGES_REQUESTED** [review]
I think uploading artifacts with results isn't needed at the moment. We will use a dashboard.

Please remove `Upload artifact` and `Upload to code-scanning` from workflow. 
Also, please remove comments related to PAT, as @evverx mentioned, we won't use it.

Thank you

**Comment by joycebrum** [comment]
Done!

**Comment by evverx** [comment]
To judge from https://github.com/systemd/systemd/actions/runs/5269007594/jobs/9526507983 it seems to be working. Thanks! It would be great if it was possible to point the badge to the dashboard here too but as far as I understand the link isn't stable yet.

LGTM.

I'll leave it to @jamacku to decide and press the button.

**Comment by evverx** [comment]
Apparently it broke Ubuntu CI somehow:
```
CONFLICT (content): Merge conflict in .github/workflows/scorecards.yml
error: could not apply 7431a54568... Remove code scanning alerts scorecards.yml
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 7431a54568... Remove code scanning alerts scorecards.yml
make[1]: *** [debian/rules:165: override_dh_auto_clean] Error 1
make[1]: Leaving directory '/tmp/autopkgtest.vYwlWR/build.PIV/systemd'
make: *** [debian/rules:295: clean] Error 2
```
@joycebrum could you squash the commits just in case?

**Comment by joycebrum** [comment]
Done, I've squashed the commits

**Review by jamacku — APPROVED** [review]
LGTM, Thank you.

Let's wait for CI.

**Pull request merged**
